### PR TITLE
conflict

### PR DESCRIPTION
**Pull request by almithani**
Just a quick lil git test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 testrepo1
-new line
```
Comment (almithani) [inline README.md:2]: this is interesting

**Review by almithani — COMMENTED** [review]
this is a general comment

File: README.md (hunk 1 above)
Comment (almithanitest1) [reply README.md:2]: I'm not sure why

**Comment by almithanitest1** [comment]
I'm not really getting what you're trying to do here

**Review by almithanitest1 — APPROVED** [review]

**Comment by almithanitest1** [comment]
maybe i kinda get it

File: README.md (hunk 1 above)
Comment (almithanitest2) [reply README.md:2]: ehh this is pretty basic

**Comment by almithanitest2** [comment]
this is a bit much

**Pull request merged**